File: C++_Programming/Practice/Program4.cpp
/*
    Algorithm

    START
        Accept First Number as No1
        Accept Second Number as No2
        Perform Addition of No1 & No2
        Display The Result

    STOP
*/

#include<iostream>
using namespace std;

int main()
{
    // Variable Creation with default values
    float i = 0.0f,j = 0.0f,k = 0.0f;

    cout<<"Enter First Number : ";
    cin>>i;

    cout<<"Enter Second Number : ";
    cin>>j;

    k = i + j;

    cout<<"Addition is : "<<k;
    
    return 0;
}

File: Java_Programming/Practice/program4.java
/*
    Algorithm

    START
        Accept First Number as No1
        Accept Second Number as No2
        Perform Addition of No1 & No2
        Display The Result

    STOP
*/

import java.util.Scanner;

class program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        // Variable Creation with default values
        int iValue1 = 0;
        int iValue2 = 0;
        int iResult = 0;

        System.out.println("Enter First Number : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter Second Number : ");
        iValue2 = sobj.nextInt();

        iResult = iValue1 + iValue2;

        System.out.println("Addition is : " + iResult);

        sobj.close();
    }
}